### PR TITLE
Delegate release notes - 791xx

### PR DESCRIPTION
**Pull request by brian-f-harness**
NG and FG Harness Delegate release notes

For reviewers, previews are available:
[NG](https://6442b59cb029d3127eb4d2d1--harness-developer.netlify.app/release-notes/delegate)
[FG ](https://6442b59cb029d3127eb4d2d1--harness-developer.netlify.app/docs/first-gen/firstgen-release-notes/fg-delegate)

### CODE REVIEW

File: docs/first-gen/firstgen-release-notes/fg-delegate.md [hunk 1]
```diff
@@ -10,6 +10,28 @@ To identify the cluster that hosts your account, open Harness FirstGen and go to
...
 
+## April 21, 2023, Harness version 791xx, Harness Delegate version 79100
+
+Harness FirstGen release 791xx includes the following feature changes and fixes for the Harness Delegate.
+
+### What's new
+
+This release introduces the following new features and enhancements:
+
+The `DELEGATE_IMAGE_TAG` version override is not considered when immutable delegates are enabled. (DEL-6202)
```
Comment (KimberlyFields) [inline docs/first-gen/firstgen-release-notes/fg-delegate.md:21]: This seems more like a fixed issue based on the description in the ticket. I don't think it belongs in What's new. Also, maybe you can work with the engineering team to determine how that change affects the customer.

File: release-notes/delegate.md [hunk 2]
```diff
@@ -15,20 +15,70 @@ Review the notes below for details about recent changes to Harness Delegate, Nex
...
 <Tabs>
   <TabItem value="What's new">
 ```
 ### What's new
 
-Added support for the latest Git CLI in the delegate maximal image. (DEL-6121) 
+This release introduces the following new features and enhancements:
 
-The latest Git CLI is now included by default. 
+- Added the following metrics for immutable delegates that you can scrapge via prometheus: (DEL-5363)
```
Comment (KimberlyFields) [inline release-notes/delegate.md:30]:
```suggestion
- Added the following metrics for immutable delegates that you can scrape via prometheus: (DEL-5363)
```

File: docs/first-gen/firstgen-release-notes/fg-delegate.md (hunk 1 above)
Comment (brian-f-harness) [reply docs/first-gen/firstgen-release-notes/fg-delegate.md]: Moved to fixed issues and will follow up with engineering.

**Review by pratmit — APPROVED** [review]

File: docs/first-gen/firstgen-release-notes/fg-delegate.md [hunk 3]
```diff
@@ -10,6 +10,28 @@ To identify the cluster that hosts your account, open Harness FirstGen and go to
 
 For FirstGen SaaS release notes, see [Harness SaaS Release Notes (FirstGen)](/docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes.md). For Self-Managed Enterprise Edition release notes, see [Self-Managed Enterprise Edition (FirstGen)](/docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md).
 
+## April 21, 2023, Harness version 791xx, Harness Delegate version 79106
+
+Harness FirstGen release 791xx includes the following feature changes and fixes for the Harness Delegate.
```
Comment (michaelcretzman) [inline docs/first-gen/firstgen-release-notes/fg-delegate.md:15]:
```suggestion
Harness FirstGen release 79111 includes the following feature changes and fixes for the Harness Delegate.
```

File: release-notes/delegate.md [hunk 4]
```diff
@@ -15,20 +15,70 @@ Review the notes below for details about recent changes to Harness Delegate, Nex
...
 <Tabs>
   <TabItem value="What's new">
 ```
 ### What's new
 
-Added support for the latest Git CLI in the delegate maximal image. (DEL-6121) 
+This release introduces the following new features and enhancements:
 
-The latest Git CLI is now included by default. 
+- Added the following metrics for immutable delegates that you can scrape via prometheus: (DEL-5363)
```
Comment (michaelcretzman) [inline release-notes/delegate.md:30]:
```suggestion
- Added the following metrics for immutable delegates that you can scrape via Prometheus: (DEL-5363)
```

File: release-notes/delegate.md [hunk 5]
```diff
@@ -46,7 +96,11 @@ This release does not include any early access features.
 
 ### Fixed issues
 
-This release does not include any fixed issues.
+This release includes the following fixes:
+
+- Added WebSocket reconnect logic when Harness Manager does not recieve a heartbeat handshake from Harness Delegate for more than five minutes. (DEL-5954)
```
Comment (michaelcretzman) [inline release-notes/delegate.md:101]:
```suggestion
- Added WebSocket reconnect logic for when the Harness Manager does not receive a heartbeat from the Harness Delegate for more than five minutes. (DEL-5954)
```

File: release-notes/delegate.md [hunk 6]
```diff
@@ -46,7 +96,11 @@ This release does not include any early access features.
...
+
+- Set the `LANG` environment variable to en_US.UTF-8 by default for delegates. (DEL-6221)
```
Comment (michaelcretzman) [inline release-notes/delegate.md:103]:
```suggestion
- Set the delegate `LANG` environment variable to en_US.UTF-8 by default. (DEL-6221)
```

File: release-notes/delegate.md [hunk 7]
```diff
@@ -58,6 +112,24 @@ This release does not include any fixed issues.
 <details>
 <summary>Expand this section to view changes to previous releases</summary>
 
+#### March 31, 2023, Harness version 78914, Harness Delegate version 78904
+
+Harness NextGen release 78914 includes the following changes for the Harness Delegate.
```
Comment (michaelcretzman) [inline release-notes/delegate.md:117]:
```suggestion
Harness release 78914 includes the following changes for the Harness Delegate.
```

Comment (michaelcretzman) [reply release-notes/delegate.md:117]: you don't need to say NextGen since these are the NG release notes.

File: release-notes/delegate.md (hunk 5 above)
Comment (michaelcretzman) [reply release-notes/delegate.md:101]: The ticket summaries we get are often not the best in terms of grammar. Always revise them for clarity, grammar, etc.

File: release-notes/delegate.md [hunk 8]
```diff
@@ -58,6 +112,24 @@ This release does not include any fixed issues.
...
+
+##### What's new
+
+- Added support for the latest Git CLI in the delegate maximal image. (DEL-6121)
+
+- The latest Git CLI is now included by default.
```
Comment (michaelcretzman) [inline release-notes/delegate.md:123]:
```suggestion
- Added support for the latest Git CLI in the delegate maximal image. (DEL-6121)
  - The latest Git CLI is now included by default.
```

Comment (michaelcretzman) [reply release-notes/delegate.md:123]: I think the second bullet is subordinate, right?

File: docs/first-gen/firstgen-release-notes/fg-delegate.md (hunk 3 above)
Comment (michaelcretzman) [reply docs/first-gen/firstgen-release-notes/fg-delegate.md:15]: Doublecheck the release number. Kim usually knows what it is but it's a last minute thing. 
I can show you how to check for NG.

**Review by michaelcretzman — DISMISSED** [review]
approved with suggestions

**Review by KimberlyFields — APPROVED** [review]

**Pull request merged**